packages/xdl/src/Android.ts

```diff
@@ -268,6 +268,12 @@ export async function openWebProjectAsync(
     await startAdbReverseAsync(projectRoot);
 
     const projectUrl = await getWebpackUrlAsync(projectRoot);  
+    if (projectUrl == null) {
```

Comment:
the == null check looks suspicious -- do you intend to catch both null & undefined? what is the intent here?

Reply:
I heard JS ninjs use `== null`